supplier/tests/test_supplier.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from conftest import TestFields
```

Comment:
You don't need to import from conftest. it is imported automatically.

Reply:
i assume the fixture methods are imported automatically but the enum classes aren't, if i delete line 2 the compiler says its not defined.